ports/lpeg/portfile.cmake

```
@@ -1,9 +1,9 @@
-set(LPEG_VER 1.0.2)
+set(LPEG_VER 1.1.0)
 
 vcpkg_download_distfile(ARCHIVE
     URLS "http://www.inf.puc-rio.br/~roberto/lpeg/lpeg-${LPEG_VER}.tar.gz"
     FILENAME "lpeg-${LPEG_VER}.tar.gz"
```

```suggestion
vcpkg_download_distfile(ARCHIVE
    URLS "https://www.inf.puc-rio.br/~roberto/lpeg/lpeg-${VERSION}.tar.gz"
    FILENAME "lpeg-${VERSION}.tar.gz"
```
`VERSION` now carries the version from the port file.